CMakeLists.txt

```diff
@@ -72,6 +72,9 @@ if (BUILDDOC)
   find_package (Doxygen)
   if (DOXYGEN_FOUND)
     configure_file (${CMAKE_CURRENT_SOURCE_DIR}/doxy/Doxyfile.in ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile @ONLY)
+    if (BUILDSWIGJAVA)
+        configure_file (${CMAKE_CURRENT_SOURCE_DIR}/doxy/Doxyfile.java.in ${CMAKE_CURRENT_BINARY_DIR}/src/java/Doxyfile @ONLY)
```

Comment:
We don't keep a /src/java in UPM. Our C++ doxyfile lives in ${CMAKE_CURRENT_BINARY_DIR} and output goes in ${CMAKE_CURRENT_BINARY_DIR}/html.
I'm thinking for java this should be ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile.java and output would go in ${CMAKE_CURRENT_BINARY_DIR}/html/java